### DISCUSSION TITLE
Locations editing and file

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2826] [Q&A]
I have Stellarium using finnish language. Location settings dialog there are button Reset Location list. Finnish translation says that it makes list empty. When I push it, the list is not empty. Iam thinking that reset would load original Location list? Is this true? If yes, I change finnish translation in Transifex to right form.

There are Delete from list button. That doesn't work. I make an issue of that.

I think that location list is huge. I don't need all them but only few near me. How can I delete the others? I have searched file where they are, but not found that in Kubuntu 22.04 operating system.

**Comment by alex-w** [comment]
You can fixed all wrong translations!

> There are Delete from list button. That doesn't work. I make an issue of that.

It works, but for user defined locations only.

> I think that location list is huge. I don't need all them but only few near me. How can I delete the others? 

You can’t do it.

**Reply by ilkant** [reply]
Delete button doen't remove a location from original list. I don't know where the list is on disk. Is that text file or binary zipped file?

**Reply by gzotti** [reply]
You cannot edit this built-in list. Seriously!
